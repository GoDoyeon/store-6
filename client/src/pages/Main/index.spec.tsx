import React from 'react';
import { render } from '@testing-library/react';
import 'jest-styled-components';

import MainPage from './index';

describe('<MainPage />', () => {
  it('should render same with snapshot', () => {
    const { container } = render(<MainPage />);
    expect(container).toMatchSnapshot();
  });
});
