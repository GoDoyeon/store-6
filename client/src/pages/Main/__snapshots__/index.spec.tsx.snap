// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainPage /> should render same with snapshot 1`] = `
.c1 {
  position: absolute;
  cursor: pointer;
  left: 20%;
  top: 110px;
}

.c2 {
  position: absolute;
  cursor: pointer;
  left: 30%;
  top: 330px;
}

.c3 {
  position: absolute;
  cursor: pointer;
  left: 10%;
  top: 580px;
}

.c4 {
  position: absolute;
  cursor: pointer;
  left: 0%;
  top: 280px;
}

.c5 {
  position: absolute;
  cursor: pointer;
  left: 70%;
  top: 240px;
}

.c6 {
  position: absolute;
  cursor: pointer;
  left: 75%;
  top: 100px;
}

.c7 {
  position: absolute;
  cursor: pointer;
  left: 65%;
  top: 550px;
}

.c8 {
  position: absolute;
  cursor: pointer;
  left: 100%;
  top: 330px;
}

.c9 {
  position: absolute;
  cursor: pointer;
  left: 90%;
  top: 550px;
}

.c0 {
  position: absolute;
  width: 100%;
  height: 815px;
  top: calc(50% - 407.5px);
}

<div>
  <div
    class="c0"
  >
    <button
      class="c1"
      type="button"
    >
      <img
        alt="icon"
        src="https://store-6-bucket.s3.ap-northeast-2.amazonaws.com/common/big-icon/book.svg"
      />
    </button>
    <button
      class="c2"
      type="button"
    >
      <img
        alt="icon"
        src="https://store-6-bucket.s3.ap-northeast-2.amazonaws.com/common/big-icon/baedal.svg"
      />
    </button>
    <button
      class="c3"
      type="button"
    >
      <img
        alt="icon"
        src="https://store-6-bucket.s3.ap-northeast-2.amazonaws.com/common/big-icon/hat.svg"
      />
    </button>
    <button
      class="c4"
      type="button"
    >
      <img
        alt="icon"
        src="https://store-6-bucket.s3.ap-northeast-2.amazonaws.com/common/big-icon/gift.svg"
      />
    </button>
    <button
      class="c5"
      type="button"
    >
      <img
        alt="icon"
        src="https://store-6-bucket.s3.ap-northeast-2.amazonaws.com/common/big-icon/house.svg"
      />
    </button>
    <button
      class="c6"
      type="button"
    >
      <img
        alt="icon"
        src="https://store-6-bucket.s3.ap-northeast-2.amazonaws.com/common/big-icon/kk.svg"
      />
    </button>
    <button
      class="c7"
      type="button"
    >
      <img
        alt="icon"
        src="https://store-6-bucket.s3.ap-northeast-2.amazonaws.com/common/big-icon/tree.svg"
      />
    </button>
    <button
      class="c8"
      type="button"
    >
      <img
        alt="icon"
        src="https://store-6-bucket.s3.ap-northeast-2.amazonaws.com/common/big-icon/pencil.svg"
      />
    </button>
    <button
      class="c9"
      type="button"
    >
      <img
        alt="icon"
        src="https://store-6-bucket.s3.ap-northeast-2.amazonaws.com/common/big-icon/colab.svg"
      />
    </button>
    <button
      style="position: absolute; left: 20%; top: 100px; z-index: -1;"
      type="button"
    >
      <img
        alt=""
      />
    </button>
    <button
      style="position: fixed; left: 50px; top: 10px; cursor: pointer;"
      type="button"
    >
      <img
        alt=""
      />
    </button>
  </div>
</div>
`;
